src/libraries/System.IO.FileSystem/tests/Directory/Copy.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Tests;
+using System.Threading;
+using Xunit;
+
+namespace System.IO.FileSystem.Tests
+{
+    public sealed class Directory_CopyDirectory : FileSystemTest
+    {
+        [Fact]
+        public void Copy()
+        {
+            const string sourceDirName = $"Test_source_{nameof(Copy)}";
+            const string destDirName = $"Test_target_{nameof(Copy)}";
```

Comment:
Don't create folders in the current working dir. as they are not automatically removed when running in CI or in someone else's PC. Use this instead.
```suggestion
            string sourceDirName = GetTestFilePath();
            string destDirName = GetTestFilePath();
```